pylint/checkers/utils.py

```diff
@@ -1245,15 +1234,15 @@ def is_none(node: nodes.NodeNG) -> bool:
     )
 
 
-def node_type(node: nodes.NodeNG) -> Optional[Any]:
+def node_type(node: nodes.NodeNG) -> Optional[nodes.NodeNG]:
```

Comment:
```suggestion
def node_type(node: nodes.NodeNG) -> Optional[type]:
```
I believe `type` was correct here.

Reply:
No worries, good to know all PR's are checked extensively!